docs/class-definitions.md

```diff
@@ -0,0 +1,59 @@
+## Class Definitions
+
+### Spaces
+
+#### constructor(ably)
+
+Create a new instance of the Spaces library.
+This requires a promise instance of the Ably library to be passed in.
+Refer to the [Ably-JS Documentation](https://github.com/ably/ably-js#introduction) for information on setting up a realtime promise client.
+
+| Property | Type                 |
+|----------|----------------------|
+| ably     | Ably.RealtimePromise |
+
+
+#### get(name, options?)
+Get or create a space. Options may only be provided if a space does not yet exist.
+
+| Property | Type                                           |
+|----------|------------------------------------------------|
+| name     | string                                         |
+| options  | [SpaceOptions](#spaceoptions) &#124; undefined |
+
+### SpaceOptions
```

Comment:
Should we add a TBD here to avoid confusion?

Reply:
Added here https://github.com/ably-labs/spaces/pull/24/commits/feac424aeb44a81c7e155e9645045a0a205c4a53

Reply:
Should this not be an object with the `offlineTimeout` option (that's what is implemented now)?